Administrator Guide/Thin-Arbiter-Volumes.md

```diff
@@ -0,0 +1,69 @@
+# Thin Arbiter volumes in gluster
+
+Thin Arbiter is a new type of quorum node where granularity
+is less compared to the traditional arbiter brick. In this
+type of volume, quorum is taken into account at a brick level.
+If there is even one file/directory which is source on a particular
+brick, the other brick will not be allowed to become source for any
+other file/directory.
+
+<!-- TOC depthFrom:1 depthTo:6 withLinks:1 updateOnSave:1 orderedList:0 -->
+
+- [Why Thin Arbiter?](#why-thin-arbiter)
+- [How Thin Arbiter works](#how-thin-arbiter-works)
+
+<!-- /TOC -->
+
+# Thin Arbiter configuration
+
+The command to run thin-arbiter process on node:
+```
+#/usr/local/sbin/glusterfsd -N --volfile-id ta-vol -f /var/lib/glusterd/vols/thin-arbiter.vol --brick-port 24007 --xlator-option ta-vol-server.transport.socket.listen-port=24007
+```
+The syntax for creating the volume is:
+```
+#glustercli volume create <volname> --replica 2 Host1:Brick-1 Host2:Brick-2 --thin-arbiter host:/path-to-store-replica-id-file ...
+```
+_**Note:** Volumes using the arbiter feature can **only** be ```replica 3 arbiter 1```_
+
+For example:
+```
+glustercli volume create testvol --replica 2 server{1..2}:/bricks/brick-{1..2} --thin-arbiter server-3:/bricks/brick_ta --force
+volume create: testvol: success: please start the volume to access data
+```
+
+
+# Why Thin Arbiter?
+This is a solution for handling stretch cluster kind of workload
+but can be used for regular workloads as well in case the users are
+satisfied with this kind of quorum more than arbiter/3-way-replication.
+Cost will be lesser than arbiter. I/O will only go to the data bricks
+and goes to thin-arbiter only in the case of first failure until heal completes.
+
+
+# How Thin Arbiter works
+There is only one process running on thin arbiter node which manages all the replica pairs across all
+volumes. Each replica pair will use its respective replica-id file that is going to be created by
```

Comment:
This is confusing to understand, the second line

Reply:
modified